### PR TITLE
[Slider]Create default light and dark color schemes for MDCSlider

### PR DESCRIPTION
**Pull request by jmdetloff**

### CODE REVIEW

File: components/Slider/src/MDCSlider.m [hunk 1]
```diff
@@ -91,20 +91,56 @@ - (void)commonMDCSliderInit {
...
   _thumbTrack.trackOffColor =
       trackBackgroundColor ? trackBackgroundColor : [[self class] defaultTrackOffColor];
-  ;
 }
 
 - (UIColor *)trackBackgroundColor {
   return _thumbTrack.trackOffColor;
 }
 
+- (void)setDisabledColor:(UIColor *)disabledColor {
```
Comment (randallli) [inline components/Slider/src/MDCSlider.m:107]: lets write a unit test for this.

File: components/Slider/src/MDCSlider.h [hunk 2]
```diff
@@ -152,6 +161,15 @@ IB_DESIGNABLE
  */
 @property(nonatomic, assign, getter=isThumbHollowAtStart) BOOL thumbHollowAtStart;
 
+/**
+ * A default color scheme for sliders displayed on light backgrounds. This color scheme is applied
+ * by default to new MDCSlider instances.
+ */
++ (nonnull MDCBasicColorScheme *)defaultSliderLightColorScheme;
```
Comment (randallli) [inline components/Slider/src/MDCSlider.h]: Can we move these into the color themer so the base component doesn't depend on the NDCBasicColorScheme?

**Review by randallli — CHANGES_REQUESTED** [review]

File: components/Slider/src/MDCSlider.m (hunk 1 above)
Comment (jmdetloff) [reply components/Slider/src/MDCSlider.m:107]: Done

File: components/Slider/src/ColorThemer/MDCSliderColorThemer.m [hunk 3]
```diff
@@ -15,13 +15,46 @@
  */
 
 #import "MDCSliderColorThemer.h"
+#import "MaterialPalettes.h"
```
Comment (randallli) [inline components/Slider/src/ColorThemer/MDCSliderColorThemer.m:18]: do we need to add a cocoapod dep?

**Review by randallli — APPROVED** [review]

File: components/Slider/src/ColorThemer/MDCSliderColorThemer.m (hunk 3 above)
Comment (jmdetloff) [reply components/Slider/src/ColorThemer/MDCSliderColorThemer.m:18]: Added!

**Pull request merged**